docs/api/1.x/synchronisation.rst

```diff
@@ -138,6 +138,11 @@ and then fetch older records in background.
 
 .. image:: ../../images/sync-both.svg
 
+.. warning::
```

Comment:
I'm not sure we should add mozilla's specific doc here (this should probably leave in syncto instead)